addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -134,6 +134,24 @@ const ThreadPatch = {
                 return raisingHandCards.concat(sessionCards, invitationCards);
             },
         });
+        this.useCameraByDefault = fields.Attr(null, {
+            compute() {
+                if (this.channel_type === "chat" && this.store.rtc.selfSession?.channel.eq(this)) {
+                    return this.store.rtc.selfSession.is_camera_on;
+                }
+                return JSON.parse(
+                    localStorage.getItem(`discuss_channel_camera_default_${this.id}`)
+                );
+            },
+            onUpdate() {
+                if (this.useCameraByDefault !== null) {
+                    localStorage.setItem(
+                        `discuss_channel_camera_default_${this.id}`,
+                        JSON.stringify(this.useCameraByDefault)
+                    );
+                }
+            },
+        });
```

Comment:
Nitpick: can `/** @this {import("models").Thread} */` above both function so the `this` is `Thread` rather than `Record`.